zapd/admin.py

```diff
@@ -18,6 +18,7 @@
 admin.add_view(UserModelView(User, db.session, category='Admin'))
 admin.add_view(RestrictedModelView(Role, db.session, category='Admin'))
 admin.add_view(RestrictedModelView(Category, db.session, category='Admin'))
+admin.add_view(AMDeviceRestrictedModelView(AMDevice, db.session, category='Admin'))
```

Comment:
We should change this from device table to wallet table, we will want to see the different wallet addresses and filter for any with more then one device

Reply:
Trying to pull the latest update from the upstream, i cant seem to pull it.

Reply:
I committed directly to your branch: https://github.com/eoliveros/zapd/commit/94691b9f597d785f5bb36fe91d966453b3fbdb9f

Reply:
after pointing to AMWallet:
![image](https://user-images.githubusercontent.com/47799652/83822441-24211d80-a725-11ea-8cac-e2724be75d28.png)


Obviously, now we need to add the other columns from the other table.

Reply:
changed the schema:
```
CREATE TABLE amdevice (
        id INTEGER NOT NULL,
        date DATETIME,
        app_version VARCHAR,
        os VARCHAR,
        os_version VARCHAR,
        manufacturer VARCHAR,
        brand VARCHAR,
        device_id VARCHAR,
        PRIMARY KEY (id)
);
CREATE TABLE amwallet (
        id INTEGER NOT NULL,
        address VARCHAR,
        amdevice_id INTEGER,
        PRIMARY KEY (id),
        FOREIGN KEY(amdevice_id) REFERENCES amdevice (id)
);
```

The amwallet.address used to be unique, but when I'm inserting the records to make it look like there's duplicate, it would fail because of "unique constraint".


The amdevice(will disable later) screen currently looks like:
![image](https://user-images.githubusercontent.com/47799652/83982278-abff6580-a979-11ea-92eb-dae15e68e8ea.png)

The amwallet looks like:
![image](https://user-images.githubusercontent.com/47799652/83982303-e5d06c00-a979-11ea-9df8-49a103209b7c.png)

will look at creating the filter.

Reply:
just pulled the latest update.
Erroring at the moment when I try to go to the AM Wallet page:

Its refering to the amdevice_list.html

Currently this page doesn't appear to exists.